tests/warp/warp_test.go

```diff
@@ -169,16 +226,21 @@ var _ = ginkgo.Describe("[Warp]", ginkgo.Ordered, func() {
 		gomega.Expect(err).Should(gomega.BeNil())
 		defer sub.Unsubscribe()
 
+		toKey, err := crypto.GenerateKey()
+		gomega.Expect(err).Should(gomega.BeNil())
+		sendToAddress = crypto.PubkeyToAddress(toKey.PublicKey)
+
 		packedInput, err := warp.PackSendWarpMessage(warp.SendWarpMessageInput{
 			DestinationChainID: blockchainIDB,
-			DestinationAddress: fundedAddress.Hash(),
+			DestinationAddress: sendToAddress.Hash(),
 			Payload:            payload,
 		})
 		gomega.Expect(err).Should(gomega.BeNil())
+
 		tx := types.NewTx(&types.DynamicFeeTx{
 			ChainID:   chainID,
-			Nonce:     0,
-			To:        &warp.Module.Address,
+			Nonce:     1,
+			To:        &chainAExAddr,
```

Comment:
why is this nonce hardcoded to 1?

Reply:
It's probably a bad idea to have it hardcoded, I think it would probably be better to fetch the nonce via the ethclient, like above